labelbox/data/annotation_types/data/raster.py

```diff
@@ -8,10 +8,44 @@
 from google.api_core import retry
 from pydantic import BaseModel
 from pydantic import root_validator
+from joblib import Memory
 
 from .base_data import BaseData
 from ..types import TypedArray
 
+memory = Memory(location="/tmp", verbose=0)
+
+
+#restructured outside of class to use memory cache
+@memory.cache
+def bytes_to_np(image_bytes: bytes) -> np.ndarray:
```

Comment:
these keys are huge. It will prob be expensive to create a hash. We know the result is unique for each object so we don't need to hash on the bytes.

Reply:
Also the cache needs to be optional